spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/JGitEnvironmentRepository.java

```diff
@@ -184,36 +103,11 @@ private synchronized Environment loadEnvironment(Git git, String application,
 		return clean(environment.findOne(application, profile, ""));
 	}
 
-	private File getWorkingDirectory() {
-		if (uri.startsWith("file:")) {
-			try {
-				return new UrlResource(StringUtils.cleanPath(uri)).getFile();
-			}
-			catch (Exception e) {
-				throw new IllegalStateException("Cannot convert uri to file: " + uri);
-			}
-		}
-		return basedir;
-	}
-
-	private String[] getSearchLocations(File dir) {
-		List<String> locations = new ArrayList<String>();
-		locations.add(dir.toURI().toString());
-		for (String path : searchPaths) {
-			File file = new File(getWorkingDirectory(), path);
-			if (file.isDirectory()) {
-				locations.add(file.toURI().toString());
-			}
-		}
-		return locations.toArray(new String[0]);
-	}
-
 	private Ref checkout(Git git, String label) throws GitAPIException {
 		CheckoutCommand checkout = git.checkout();
 		if (shouldTrack(git, label)) {
 			trackBranch(git, checkout, label);
-		}
-		else {
+		} else {
```

Comment:
Try and keep our [code conventions](https://github.com/spring-cloud/spring-cloud-netflix/#code-conventions-and-housekeeping)